docs/resources/aks_cluster_v3.md

```diff
@@ -0,0 +1,541 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "rafay_aks_cluster_v3 Resource - terraform-provider-rafay"
+subcategory: ""
+description: |-
+  
+---
+
+# rafay_aks_cluster_v3 (Resource)
+
+This example is for the AKS Cluster v3.
+
+## Example Usage
+
+```terraform
+resource "rafay_aks_cluster_v3" "demo-terraform" {
+  metadata {
+    name    = "aks-v3-tf-1"
+    project = "defaultproject"
+  }
+  spec {
+    type          = "aks"
+    blueprint_config {
+      name = "default-aks"
+      version = "1.21.0"
+    }
+    cloud_credentials = "aks-cred"
+    config {
+      kind       = "aksClusterConfig"
+      metadata {
+        name = "aks-v3-tf-1"
+        project = "defaultproject"
```

Comment:
config.metadata.project field is incorrect. please remove it from the schema. We need to give the project details once which we provided in cluster metadata